src/sentry/static/sentry/app/views/discover/result/utils.tsx

```diff
@@ -366,7 +366,11 @@ export function downloadAsCsv(result: SnubaResult) {
     }),
   });
 
-  const encodedDataUrl = encodeURI(`data:text/csv;charset=utf8,${csvContent}`);
+  // Need to also manually replace # since encodeURI skips them
```

Comment:
Probably shouldn't be touching v1, but figured I might as well fix it for the few weeks its still around.